src/assets/stylesheets/base/_buttons.scss

```diff
@@ -0,0 +1,8 @@
+button {
+  border: transparent;
+
+  background-color: map-get($brand-white, 10);
+  
+  color: #15151a;
+  font-weight: 800;
```

Comment:
it's better to get font-weight from variables as well, i.e. 

```
$font-weight: (
  normal: 400,
  bolder: 800
);

```

Reply:
Fixed in 4bef81c390a352cc9ea3b6cf06234379828d3cbc